pkgs/development/compilers/llvm/7/default.nix

```diff
@@ -87,48 +79,78 @@ let
 
     lldb = callPackage ./lldb.nix {};
 
+    # Below, is the LLVM bootstrapping logic. It handles building a
```

Comment:
Hmm, why does this go in LLVM 7 and not the others?

Reply:
It was already in the others,

Reply:
ahhhh of course, sorry for the noise.

Reply:
no worries!